server.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
+var express = require('express');
+var app = express();
+var PORT = process.env.PORT || 8080
+
+app.use(express.static(path.join(__dirname, 'dist')));
+
+app.get('/', function(request, response) {
```

Comment:
Maybe put a comment above the first `function(request, response)` saying that:

```
/*
*  No need for an arrow function here since we do not need to bind this
*/
```